cirq/ops/common_channels.py

```diff
@@ -31,10 +31,20 @@
 class AsymmetricDepolarizingChannel(gate_features.SingleQubitGate):
     """A channel that depolarizes asymmetrically along different directions."""
 
-    def __init__(self, p_x: float, p_y: float, p_z: float) -> None:
+    def __init__(self, *args, **kwargs) -> None:
         r"""The asymmetric depolarizing channel.
 
         This channel applies one of four disjoint possibilities: nothing (the
+        identity channel) or one of the 2**n pauli gates. The disjoint
```

Comment:
The number of Pauli strings on n qubits is 4\*\*n, not 2\*\*n. This figure includes the identity.

Reply:
Done.